sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -209,8 +239,16 @@ case class Multiply(left: Expression, right: Expression)
 }
 
 @ExpressionDescription(
-  usage = "a _FUNC_ b - Divides a by b.",
-  extended = "> SELECT 3 _FUNC_ 2;\n 1.5")
+  usage = "expr1 _FUNC_ expr2 - Divides expr1 by expr2.",
```

Comment:
Worth saying that this is floating point division? I guess the example clarifies that. Above you write "expr1+expr2" but here you say "divides expr1 by expr2". Might be worth staying consistent.